llvm/test/CodeGen/AArch64/neon-partial-reduce-dot-product.ll

```diff
@@ -649,69 +774,93 @@ define <4 x i32> @udot_no_bin_op(<4 x i32> %acc, <16 x i8> %a){
 define <4 x i32> @udot_no_bin_op_in_loop(ptr %p){
```

Comment:
These check statements look identical to me, so they could share a `CHECK` prefix.